Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -43,23 +50,42 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
     {
         super.refresh(significantChange);
 
-        let fontMap = this.nodeStyles.computedPrimaryFont?.calculateFontProperties(this.nodeStyles) ?? {};
-        this._fontPropertiesMap = fontMap.propertiesMap ?? new Map;
-        this._fontVariationsMap = fontMap.variationsMap ?? new Map;
-        this._fontFeaturesMap = fontMap.featuresMap ?? new Map;
+        // FIXME: Prevent refreshing FontStyles if we're the ones causing the changes.
```

Comment:
Ditto about creating a bugzilla and referencing it here.

Reply:
Filed and noted: https://bugs.webkit.org/show_bug.cgi?id=250128